content/en/docs/components/training/getting-started.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Getting Started"
+description = "Get started with Training Operator"
+weight = 30
++++
+
+This guide describes how to get started with Training Operator and run a few simple examples.
+
+## Prerequisites
+
+You need to install the following components to run examples:
+
+- Training Operator [installed](/docs/components/training/installation/#installing-training-operator).
+- Training Python SDK [installed](/docs/components/training/installation/#installing-training-python-sdk).
+
+## Getting Started with PyTorchJob
+
+You can create your first Training Operator distributed PyTorchJob using Python SDK. Define the
+training function that implements end-to-end model training. Each Worker will execute this
+function on the appropriate Kubernetes Pod. Usually, this function contains logic to
+download dataset, create model, and train the model.
+
+Training Operator will automatically set `WORLD_SIZE` and `RANK` for the appropriate PyTorchJob
+worker to perform [PyTorch Distributed Data Parallel (DDP)](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).
+
+For example:
+
+```python
+def train_func():
+    import torch
+    import os
+
+    # Create model.
+    class Net(torch.nn.Module):
+        """Create the Pytorch model"""
+        ...
```

Comment:
nit: my recommendation would be to populate this even with a trivial single layer NN that would actually run for this example. It helps users get started that may just be executing copy-pasta.

Reply:
Sure, let me try to add something simple.